packages/account/src/Components/self-exclusion/self-exclusion-article-content.tsx

```diff
@@ -100,6 +100,7 @@ export const selfExclusionArticleItems = ({
 
     const getNonEuItems = () => [
         {
+            key: 0,
```

Comment:
@jim-deriv Are we adding all this key only to get rid of a warning about using indexes as keys in self-exclusion-article-content.tsx ? :) Good trick but basically, it doesn't change anything because indexes have the same values as these keys lol. And btw SonarCloud contradict itself, it gives reference to [React API](https://legacy.reactjs.org/docs/reconciliation.html#recursing-on-children) which says literally the following: As a last resort, you can pass an item’s index in the array as a key. This can work well if the items are never reordered. In our case these items are never re-ordered. Just sharing thoughts out loud :)

Reply:
@maryia-deriv, I recall making the same point as you in one of the comments in [this](https://github.com/binary-com/deriv-app/pull/6048) PR, but it seemed like the use of index was still not desired. I'll update the code to use index like before. 🙂